packages/grafana-ui/src/components/Table/utils.ts

```diff
@@ -57,6 +61,7 @@ export function getColumns(data: DataFrame, availableWidth: number, columnMinWid
 
     columns.push({
       Cell,
+      id: field.name,
```

Comment:
field name is not guaranteed to be unique sadly (A table can include multiple fields with same name). Can we use index instead?

Reply:
Great catch, I'll do that instead!

Reply:
So I tried several approaches, but no one worked. 
- Adding index as id, react-table will replace the id with the Header name
- Using the accessor to retrieve the field failed too because it didn't match the correct memory area, got [object, object] instead of real values.
- Using the accessor to retrieve the index of field failed too for unknown reason, got strange values instead of real values.

So I settled with just using index from the loops, which seem to work for now when we don't have reordering of columns. I really don't like the react-table API because I'm just not getting it at all.